src/lib/components/FileManager/FileManager.js

```diff
@@ -0,0 +1,78 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Dropzone from 'react-dropzone';
+import { FileArea } from './FileArea';
+import { UploadState } from '../../state/reducers/files';
+
+export default class FileManager extends Component {
```

Comment:
another silly one: it sounds weird to have a cmp called "FileManager" :) Is it more about the component that renders the uploaded files right and the dropzone right?

Reply:
Did some refactoring on the name and merged `FileManager` and `FileArea` in one `FileUploader` file. What do you think?